.github/workflows/release.yml

```
@@ -49,6 +50,12 @@ jobs:
           cmake --build ${{github.workspace}}/build --config Release
           cpack --config ${{github.workspace}}/build/CPackConfig.cmake -C Release -G ZIP
 
+      - name: Test  # don't release unless tests pass
+        working-directory: ${{github.workspace}}/build
+        run: |
+          ctest -C Release -L anyplatform --output-on-failure
+          ctest -C Release -L acquire-driver-egrabber --output-on-failure
```

You can combine the ctest labels into one. This way all the tests run even if one fails early.

```suggestion
        run: ctest -C Release -L anyplatform -L acquire-driver-egrabber --output-on-failure
```